CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.4
```

Comment:
```suggestion
## 0.4.0
```

to be precise with semver, despite previous version listed in Changelog is `0.3`

Reply:
Very true, correct this one and the others.